metadata-ingestion/docs/sources/sigma/sigma_pre.md

```diff
@@ -20,4 +20,55 @@ This source extracts the following:
 | `Page`                 | [Dashboard](../../metamodel/entities/dashboard.md)            |                                  |
 | `Element`              | [Chart](../../metamodel/entities/chart.md)                    |                                  |
 | `Dataset`              | [Dataset](../../metamodel/entities/dataset.md)                | SubType `"Sigma Dataset"`        |
-| `User`                 | [User (a.k.a CorpUser)](../../metamodel/entities/corpuser.md) | Optionally Extracted             |
\ No newline at end of file
+| `User`                 | [User (a.k.a CorpUser)](../../metamodel/entities/corpuser.md) | Optionally Extracted             |
+
+## Advanced Configurations
+
+### Chart source platform mapping
+If you want to provide platform details(platform name, platform instance and env) for chart's all external upstream data sources, then you can use `chart_sources_platform_mapping` as below:
+
+#### Example - For just one specific chart's external upstream data sources
+```yml
+    chart_sources_platform_mapping:
+      'workspace_name/workbook_name/chart_name_1': 
+        data_source_platform: snowflake
+        platform_instance: new_instance
+        env: PROD
+
+      'workspace_name/folder_name/workbook_name/chart_name_2': 
+        data_source_platform: postgres
+        platform_instance: cloud_instance
+        env: DEV
+```
+
+#### Example - For all charts within one specific workbook
+```yml
+    chart_sources_platform_mapping:
+      'workspace_name/workbook_name_1': 
+        data_source_platform: snowflake
+        platform_instance: new_instance
+        env: PROD
+      
+      'workspace_name/folder_name/workbook_name_2': 
+        data_source_platform: snowflake
+        platform_instance: new_instance
+        env: PROD
+```
+
+#### Example - For all workbooks charts within one specific workspace
+```yml
+    chart_sources_platform_mapping:
+      'workspace_name': 
+        data_source_platform: snowflake
+        platform_instance: new_instance
+        env: PROD
+```
+
+#### Example - For all ingested charts
```

Comment:
follow up - the order of these examples should be reversed, and they should be rephrased as below

```suggestion
#### Example - All workbooks use the same connection
```